### DISCUSSION TITLE
2023/09/27/creating-my-blog-with-jekyll-github-pages-giscus-and-zapier

### DISCUSSION THREAD

**Discussion by giscus** [TeriyakiGod/blog#8] [Announcements]
# 2023/09/27/creating-my-blog-with-jekyll-github-pages-giscus-and-zapier

Hi! Thanks for visiting my blog where I share the progress of my computer engineering journey. Enjoy!

https://blog.kacperochnik.eu/2023/09/27/creating-my-blog-with-jekyll-github-pages-giscus-and-zapier.html

**Comment by TeriyakiGod** [comment]
Hey! Share your experience with the tutorial down here in the comments!
